projects/samples/howto/docker/simulation/docker-compose-webots.yml

```
@@ -0,0 +1,25 @@
+version: "3.8"
+
+volumes:
+  tmp:
+    name: tmp-$PORT-$ROBOT_NAME_1
+
+services:
+  webots:
+    image: simulation
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - WEBOTS_DEFAULT_IMAGE=$IMAGE
+    command: /usr/local/webots/webots --port=$PORT --mode=realtime --heartbeat=5000 --extern-urls --batch $WORLD
+    environment:
+      - DISPLAY=:0
```

```suggestion
      - DISPLAY
```
I had to make this change to make it work on my computer as I have `DISPLAY=1` by default.
It would be good if someone with `DISPLAY=0` could test the fix to see if it still works for them.

---

I just tested and I confirm this works. My `DISPLAY` environment variable is set to `:0`.